docs/content/preview/manage/backup-restore/snapshot-ysql.md

```diff
@@ -132,30 +154,28 @@ the folders of ONLY the leader tablets on that node. Because each tablet-replica
 
     {{< /note >}}
 
-Your snapshot of the YSQL database is complete.
+8. If you don't want to keep the in-cluster snapshot, it's now safe to [delete it](#deleting-a-snapshot).
 
------
-
-## Restore a snapshot
+## Restoring a Snapshot from an External Storage
 
-Let’s destroy the existing cluster, create a new cluster, and import the snapshot that we had previously created.
+To restore a snapshot that had been [moved to an external storage](#moving-a-snapshot-to-an-external-storage), go through the steps below.
```

Comment:
After you've cleaned up all the headings, make sure that all the anchor links still work...
```suggestion
To restore a snapshot that you've [moved to external storage](#move-a-snapshot-to-external-storage), do the following:
```